setup.py

```diff
@@ -16,4 +16,6 @@
     include_package_data=True,
     install_requires=requirements,
     entry_points={'console_scripts': ['pyplane=pyplane.app:run']},
+    data_file = [
```

Comment:
Is this necessary? Currently it seems a bit pointless but maybe there is some good reason behind this empty list.

If so an explanatory comment would help.